tests/zfs-tests/tests/functional/migration/setup.ksh

```diff
@@ -57,7 +57,11 @@ log_must zfs set mountpoint=$TESTDIR $TESTPOOL/$TESTFS
 rm -rf $NONZFS_TESTDIR  || log_unresolved Could not remove $NONZFS_TESTDIR
 mkdir -p $NONZFS_TESTDIR || log_unresolved Could not create $NONZFS_TESTDIR
 
-echo "y" | newfs -v ${DEV_DSKDIR}/$NONZFS_DISK
+if is_freebsd; then
+	newfs ${DEV_DSKDIR}/$NONZFS_DISK
+else
+	echo "y" | newfs -v ${DEV_DSKDIR}/$NONZFS_DISK
+fi
```

Comment:
Again, I think creating a wrapper function for this would be worthwhile.  The `echo "y"` syntax is something we inherited from Illumos and should be replaceable with `-F` if needed.